packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -121,11 +121,14 @@ class ChartState extends State<Chart> with AutoDisposeMixin {
   }
 }
 
+/// Used to track current and previous x,y position and where the bottom
+/// of the Y is located (for non stacked traces yBottom is 0).
 class Point {
```

Comment:
nit: can we rename this class from Point now that it has a yBottoma as well as the more typical x and y.

Reply:
Renamed to PointAndBase.